src/Lean/PrettyPrinter/Delaborator/Options.lean

```diff
@@ -90,6 +90,11 @@ register_builtin_option pp.safeShadowing  : Bool := {
   group    := "pp"
   descr    := "(pretty printer) allow variable shadowing if there is no collision"
 }
+register_builtin_option pp.tagSymbols : Bool := {
```

Comment:
The name here is a bit misleading, the option does not only affect symbols (which I associate with `+`, etc.) but any head symbol (such as `f` in `f a b`).  The terminology isn't great, we have lots of things that are called symbols.

Reply:
Maybe `tagAllFunctions` ?

Reply:
The `All` seems redundant. How about `tagAppFns`, in reference to `Expr.getAppFn`?

Reply:
Would also be good to have a comment that if the constant is not part of an application, the tagging is unconditionally done by `delabFor`.